examples/benchmark/benchmark_detectors.py

```diff
@@ -19,7 +19,7 @@
     print('Usage: python benchmark_detectors.py <benchmark>')
     exit()
 
-with open('config.yml') as config_file:
+with open(Path(__file__).parent / 'config.yml') as config_file:
```

Comment:
Why is this necessary?

Reply:
in case someone calls the benchmark script from somewhere else than the `examples/benchmark` folder... feel free to revert if you think that's not necessary :)

Reply:
I don't think this is necessary, because in our instructions we explicitly mention to change into the directory first.